src/components/PupilComponents/QuizOrderAnswer/QuizOrderAnswer.tsx

```diff
@@ -0,0 +1,130 @@
+
+import { useQuizEngineContext } from "@/components/PupilComponents/QuizEngineProvider";
+import { useInitialChange } from "@/components/PupilComponents/QuizUtils/useInitialChange";
+import OakError from "@/errors/OakError";
+
+export type QuizOrderAnswerProps = {
+  onInitialChange?: () => void;
+  onChange?: () => void;
+};
+
+export const QuizOrderAnswer = ({
+  onChange,
+  onInitialChange,
+}: QuizOrderAnswerProps) => {
+  const { handleOnChange: handleInitialChange } = useInitialChange({
+    onChange,
+    onInitialChange,
+  });
+  const { currentQuestionData, questionState, currentQuestionIndex } =
+    useQuizEngineContext();
+  invariant(currentQuestionData, "currentQuestionData is not defined");
+  const answers = currentQuestionData.answers;
+  const questionUid = currentQuestionData.questionUid;
+  const feedback = questionState[currentQuestionIndex]?.feedback;
+
+  invariant(
+    answers && isOrderAnswer(answers),
+    `current '${questionUid}' is not an order question`,
+  );
+
+  /**
+   * Memoise the randomised order of items to preserve it across renders
+   */
+  const initialItems = useMemo(() => {
+    const originalItems = answers.order.map((item, index) => {
+      const label = item?.answer?.[0]?.text;
+      invariant(
+        label,
+        `label is missing for option in question '${questionUid}'`,
+      );
+
+      return {
+        id: (index + 1).toString(),
+        label,
+      };
+    });
+    let currentItems = originalItems.slice();
+
+    // Randomise items while ensuring that the random order is not the same as the original order
+    while (
+      originalItems.length > 1 &&
+      JSON.stringify(originalItems) === JSON.stringify(currentItems)
+    ) {
+      currentItems = currentItems.sort(() => 0.5 - Math.random());
+    }
+
+    return currentItems;
+  }, [answers.order, questionUid]);
+  const [currentOrder, setCurrentOrder] = useState(initialItems);
+
+  const handleOrderChange = (items: OakQuizOrderProps["initialItems"]) => {
+    handleInitialChange();
+    setCurrentOrder(items);
+  };
+
+  if (feedback) {
+    invariant(isArray(feedback), "question feedback is not an array");
+
+    return (
+      <OakBox>
+        {currentOrder.map((item, i) => {
+          const currentFeedback = feedback.at(i);
+          invariant(currentFeedback, "feedback is missing");
+
+          return (
+            <OakDroppable key={item.id} $mb="space-between-xs">
+              <OakDraggableFeedback
+                feedback={currentFeedback}
+                data-testid="order-item-feedback"
+              >
+                {item.label}
+              </OakDraggableFeedback>
+            </OakDroppable>
+          );
+        })}
+      </OakBox>
+    );
+  }
+
+  return (
+    <OakBox>
+      <OakQuizOrder initialItems={initialItems} onChange={handleOrderChange} />
+      {currentOrder.map((item) => {
+        return (
+          <input
+            key={item.id}
```

Comment:
This is the bit i couldnt get my head around when trying to implement; i see youve decoupled the draggable ui component from the input tags; and add them back in for the functionality so you can use FormData object. 

is there a reason why we dont use OakInput component here?